src/components/CreditCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+function CreditCard(props) {
+    let { type, number, expirationMonth, expirationYear, bank, owner, bgColor, color } = props;
+    const last4Str = number.slice(-4);
+    return (
+        <div className='creditcard'>
+
+            <div className='bgCard' style={{ backgroundColor: `${bgColor}`, color: `${color}` }}>
+
+                <img src="/images/visa.png" alt="" />
```

Comment:
In React, to be able to use an image, you have to import it first on the top of the file.